packages/frontend/styles/global.css

```diff
@@ -14,6 +14,7 @@
   --card-bg: #fcfcfc;
   --default-button: #39baba;
   --shadow: 0 0 24px rgba(155, 201, 255, 0.24);
+  --label-color: #b4b4b4;
```

Comment:
@tolgaduzenli we already have variable called --label-color with same hex color.